packages/blockchain-wallet-v4-frontend/src/components/Send/ExchangePromo/index.tsx

```diff
@@ -60,7 +60,7 @@ type Props = LinkStatePropsType & LinkDispatchPropsType
 class ExchangePromo extends PureComponent<Props> {
   onSignup = () => {
     this.props.modalActions.closeAllModals()
-    this.props.modalActions.showModal('LinkToExchangeAccount', {
+    this.props.modalActions.showModal('LINK_TO_EXCHANGE_ACCOUNT_MODAL', {
```

Comment:
Asking for switching to enums is probably a huge task right?

Reply:
I tried, but it will take more than 2 days to do it.

Reply:
Yeah not need to be as part of this one, we might open a ticket for it in backlog ;)